.github/CONTRIBUTING.md

```diff
@@ -55,119 +136,3 @@ Also, if necessary, you can specify custom HTTP User-Agent or Referrer via `#EXT
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64)
 http://example.com/stream.m3u8
 ```
-
```

Comment:
Why remove this section?

Reply:
Moved the remaining sections to the new version.